### PR TITLE
Improve error message when player is softbanned and try to use berry

### PR DESCRIPTION
**Pull request by earthchie**
Instead of show Error code: 1, inform player that they're softbanned instead.

### CODE REVIEW

**Comment by surfaace** [comment]
please resolve build errors

File: pokemongo_bot/logger.py [hunk 1]
```diff
@@ -9,10 +9,9 @@
 
 def log(string, color = 'white'):
     colorHex = {
-        'red': '91m',
         'green': '92m',
         'yellow': '93m',
-        'blue': '94m'
```
Comment (dmateusp) [inline pokemongo_bot/logger.py]: you took off the blue color here, is it a mistake ?

File: pokemongo_bot/cell_workers/pokemon_catch_worker.py [hunk 2]
```diff
@@ -149,24 +126,23 @@ def work(self):
...
+                                    logger.log(
+                                        '[x] Captured {}! [CP {}] [IV {}]'.format(
+                                            pokemon_name,
+                                            cp,
+                                            pokemon_potential
+                                        ), 'green'
+                                    )
+
                                     id_list2 = self.count_pokemon_inventory()
-
```
Comment (dmateusp) [inline pokemongo_bot/cell_workers/pokemon_catch_worker.py]: you deleted changes on Captured pokemons messages that are not related to your PR

**Comment by earthchie** [comment]
sorry. I forget about this PR. I tried to figure out this issue: https://github.com/PokemonGoF/PokemonGo-Bot/issues/765

please close this for now until above issue fixed
